test/Infrastructure/LeanCode.ForceUpdate.Tests/ForceUpdateTests.cs

```diff
@@ -0,0 +1,118 @@
+using System.Net;
+using System.Text.Json;
+using LeanCode.ForceUpdate.Contracts;
+using FluentAssertions;
+using Xunit;
+
+namespace LeanCode.ForceUpdate.Tests;
+
+public class ForceUpdateTests : ForceUpdateTestsBase
+{
+    [Fact]
+    public async Task Versions_correctly_returns_all_versions()
```

Comment:
My previous comment still holds - you don't need the whole asp.net pipeline here, calling QH directly would be far less awkward.

My proposition is to build a container, resolve the handler and work on it, like this:
```
var services = new ServiceCollection();
services.AddCQRS().AddForceUpdate()
(register configs)

var sp = services.BuildServiceProvider();
var qh = services.GetRequiredService<QH>()
```
Then you can test di registration and handler easily.

One more nit: if you were to keep the full http pipeline, I'd pack the call in better method - there's no need to specify path and raw json content in every tests.